packages/spectacular/src/lib/feature-testing/configuration/spectacular-feature-testing-features.ts

```diff
@@ -0,0 +1,14 @@
+import type { Provider } from '@angular/core';
+
+export type SpectacularFeatureTestingFeatureKind = 'InitialFeatureNavigation';
+
+export interface SpectacularFeatureTestingFeature<
+  TFeatureKind extends SpectacularFeatureTestingFeatureKind
+> {
+  readonly ɵbrand: 'SpectacularFeatureTestingFeature';
+  readonly ɵkind: TFeatureKind;
+  readonly ɵproviders: Provider[];
```

Comment:
question: I remember that you mentioned that the `ɵ` was unstable and prone to changes. Is this different?

Reply:
In Angular packages, it means *internal* but can also mean experimental/unstable. Here, I'm following the pattern of Angular HTTP and Angular Router tree-shakable feature types to support passing Angular Router features to `provideSpectacularFeatureTesting`. I'm going to document these types before this PR is merged.

Reply:
These properties have been marked as `@internal` so they won't show up in the API reference and this annotation will be visible to library consumers navigating the type definitions.